### PR TITLE
Added gemmer

### PR DESCRIPTION
**Pull request by captainbuckkets**
Please review for merge

### CODE REVIEW

**Comment by peerchemist** [comment]
Gemmer is just a mobile wallet, not both mobile/web.

**Comment by bananenwilly** [comment]
@captainbuckkets please elaborate on why you're putting english strings into French/German translation files. 
English will always serve as backup and does not need to be added to all loc files.

**Review by bananenwilly — CHANGES_REQUESTED** [review]
clean up translation files
consider commit from chemist

**Comment by captainbuckkets** [comment]
> @captainbuckkets please elaborate on why you're putting english strings into French/German translation files. 
> English will always serve as backup and does not need to be added to all loc files. 
> 

Because I don't know what I'm doing since it moved to React. Been a whole year

**Comment by bananenwilly** [comment]
> 
> 
> > @captainbuckkets please elaborate on why you're putting english strings into French/German translation files.
> > English will always serve as backup and does not need to be added to all loc files.
> 
> Because I don't know what I'm doing since it moved to React. Been a whole year

Whoever chose to do it this way left the rest of us with a massive headache.
See https://github.com/peercoin/peercoin.net/pull/166#issuecomment-767650737 for example

**Comment by captainbuckkets** [comment]
I miss HTML3. I'll try to look tomorrow of i have time

**Comment by bananenwilly** [comment]
> 
> 
> I miss HTML3. I'll try to look tomorrow if I have tim

**Comment by bananenwilly** [comment]
oops

**Comment by peerchemist** [comment]
Link to gemmer should point to the Play Store.

**Review by captainbuckkets — COMMENTED** [review]
I didn't see EN strings going into DE/FR but did this solve the problem?

File: public/locales/de/translation.json [hunk 1]
```diff
@@ -321,7 +321,8 @@
     "paperWalletSubtitle2": "Unofficial Wallets",
     "type1": "Hardware Wallet",
     "type2": "Mobile/Web Wallet",
-    "type3": "Web Wallet"
+    "type3": "Web Wallet",
+    "type4": "Mobile Wallet"
```
Comment (bananenwilly) [inline public/locales/de/translation.json:325]: ^^

File: public/locales/en-US/translation.json [hunk 2]
```diff
@@ -323,7 +323,8 @@
     "paperWalletSubtitle2": "Unofficial Wallets",
     "type1": "Hardware Wallet",
     "type2": "Mobile/Web Wallet",
-    "type3": "Web Wallet"
+    "type3": "Web Wallet",
+    "type4": "Mobile Wallet"
```
Comment (bananenwilly) [inline public/locales/en-US/translation.json:327]: ^^^

File: public/locales/fr/translation.json [hunk 3]
```diff
@@ -299,7 +299,8 @@
     "paperWalletSubtitle2": "Unofficial Wallets",
     "type1": "Hardware Wallet",
     "type2": "Mobile/Web Wallet",
-    "type3": "Web Wallet"
+    "type3": "Web Wallet",
+    "type4": "Mobile Wallet"
```
Comment (bananenwilly) [inline public/locales/fr/translation.json:303]: ^^^

**Review by captainbuckkets — COMMENTED** [review]
Updated

**Review by bananenwilly — APPROVED** [review]

**Pull request merged**